src/geos_types.jl

```diff
@@ -1,5 +1,35 @@
 abstract type AbstractGeometry end
 
+function get_context(gs::AbstractVector)::GEOSContext
```

Comment:
Can we provide a docstring for `get_context(...)`? E.g. returns the context that is shared among all the geometries in `gs`. It returns the global context if `gs` is empty, and raises an error if any of the geometries in `gs` have a different context.

Reply:
done!